contrib/kind.yaml.j2

```diff
@@ -5,8 +5,6 @@ networking:
   kubeProxyMode: "none"
   # the default CNI will not be installed
   disableDefaultCNI: true
-  apiServerAddress: {{ ovn_apiServerAddress | default('0.0.0.0') }}
```

Comment:
removing all the apiServerAddress stuff needs some explanation in the commit message

Reply:
Will do, just going off what KIND recommends -> https://kind.sigs.k8s.io/docs/user/configuration/#api-server

Reply:
After talking to Antonio, he said the only reason we did this originally was actually for you :) so that you could access the API running on a remote server?

Reply:
:man_shrugging: haha OK. I don't need to do that anymore. Maybe I was trying to run on a remote server. We can always revert it later if someone complains.